scripts/main.py

```diff
@@ -16,7 +16,6 @@
 import argparse
 from logger import logger
 import logging
-from prompt import get_prompt
```

Comment:
While you are at it, revert this change as well as unrelated. You can submit it as a separate PR. We are happy to merge one-liners.